code/datums/diseases/transformation.dm

```diff
@@ -39,6 +39,8 @@
 	if(istype(affected_mob, /mob/living/carbon) && affected_mob.stat != DEAD)
 		if(stage5)
 			to_chat(affected_mob, pick(stage5))
+		if(name == "Gluttony's Blessing")
```

Comment:
This is quite snowflaky.
It would be better to add a `transformation_text` variable or such. Which if not null will be displayed to the user.
This is also displayed before the `jobban_isbanned` is done. Which could lead to the message being shown while the user not becoming the mob